scripts/setup-adapters.sh

```diff
@@ -46,6 +46,18 @@ function install_libhdfs3 {
   cmake_install
 }
 
+function install_protobuf {
+  yum -y install gcc-c++
```

Comment:
Is this required to be part of this script? This should be part of the environmental setup?

Reply:
> Is this required to be part of this script? This should be part of the environmental setup?

It is necessary, we need install gcc-c++ to compile protobuf. we will get error when we compile protobuf without gcc-c++.